daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/PackageInterface.scala

```diff
@@ -317,7 +317,17 @@ private[lf] class PackageInterface(signatures: PartialFunction[PackageId, Packag
       case Right(interface) =>
         interface.fixedChoices.get(chName) match {
           case Some(choice) => Right(ChoiceInfo.Interface(choice))
-          case None => Left(LookupError(context, context))
+          case None => {
+            interface.requires.view
```

Comment:
Can you add a TODO to improve this lookup, please, and link it to the issue https://github.com/digital-asset/daml/issues/11345

Reply:
done.